readthedocs/subscriptions/tests/test_event_handlers.py

```diff
@@ -430,16 +430,3 @@ def test_subscription_precedence(self):
 
             self.organization.refresh_from_db()
             assert self.organization.stripe_subscription == stripe_subscription
-
-    def test_register_events(self):
```

Comment:
Doesn't look like we should delete these?

Reply:
I deleted it because "webhooks.registration" doesn't exist anymore. They deleted that file completely from the repository and I didn't find a replacement 🤷‍♂️